### PR TITLE
Fix unbound this in WalletLinkConnector

### PR DESCRIPTION
**Pull request by brendanww**
Updates the WalletLink connector to fix issue from unbound `this`.

### CODE REVIEW

File: example/connectors.ts [hunk 1]
```diff
@@ -26,14 +26,15 @@ export const network = new NetworkConnector({
 })
 
 export const walletconnect = new WalletConnectConnector({
-  rpc: { 1: RPC_URLS[1], 4: RPC_URLS[4] },
```
Comment (brendanww) [inline example/connectors.ts:29]: example app was throwing an error about WalletConnect not liking multiple rpc url's.

File: packages/walletlink-connector/src/index.ts [hunk 2]
```diff
@@ -73,7 +77,7 @@ export class WalletLinkConnector extends AbstractConnector {
     if (__DEV__) {
       console.log("Handling 'chainChanged' event with payload", chainId)
     }
-    this.emitUpdate({ chainId })
+    this.emitUpdate({ chainId: chainId })
```
Comment (petejkim) [inline packages/walletlink-connector/src/index.ts:80]: why was this changed?

Comment (brendanww) [reply packages/walletlink-connector/src/index.ts:80]: to be explicit; seems less fragile

File: packages/walletlink-connector/src/index.ts [hunk 3]
```diff
@@ -7,7 +7,8 @@ interface WalletLinkConnectorArguments {
   url: string
   appName: string
   appLogoUrl?: string
-  darkMode?: boolean
+  darkMode?: boolean,
```
Comment (NoahZinsmeister) [inline packages/walletlink-connector/src/index.ts:10]:
```suggestion
  darkMode?: boolean
```
